### ISSUE TITLE
[Playground]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for '@beta' text in 'Playground' page.[Sev2][WCAG1.4.3][Auto]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#14]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Navigate to 'Playground' link and activate it.
4) Now navigate to 'Choose a sample' combo box and activate it.
5) Select ' A Basic example' from the listed menu items.
6) Now run Accessibility Insights for web tool and observe the issue.

**Actual:**
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for '@beta' text present in 'A Basic example' menu item in 'Playground' page. This element has insufficient contrast ratio of 4.33.

**Expected:**
Contrast ratio for the text must be greater than or equal to 4.5:1

**User Impact:**
When a minimum contrast ratio is not provided between text and its background, it is difficult for low-vision users as well as anyone in extreme lighting environments to read the content.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum
<img width="960" alt="Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for '@beta' text in 'Playground' page" src="https://user-images.githubusercontent.com/85154417/121009312-a2d5ee00-c7b1-11eb-8b6a-e8a889bb9fd3.png">

**Comment by apostolisms** [comment]
[This](https://github.com/microsoft/tsdoc/pull/303) PR fixes the issue.
